mlflow/metrics/genai/prompts/v1.py

```diff
@@ -45,9 +45,9 @@
 
 {examples}
 
-You must return the following fields in your response one below the other:
+You must return the following fields in your response in two lines, one below the other. Do not add additional new lines.
 score: Your numerical score for the model's {name} based on the rubric
-justification: Your step-by-step reasoning about the model's {name} score
```

Comment:
removing this as this performs poorly with llama2. for example,
```
score: 3
justification:

1. The input asks, "What is MLflow?"
2. The output responds, "MLflow is an open-source package."
3. The additional information used by the model mentions that MLflow is an open-source platform for managing the end-to-end machine learning (ML) lifecycle and was developed by Databricks, a company that specializes in big data and machine learning solutions.
4. The output does not directly address the question asked in the input, but it does provide some relevance to the question by mentioning that MLflow is an open-source package, which is related to the context provided.
5. Therefore, based on the grading rubric, a score of 3 is appropriate, as the output provides some relevance to the question and is somehow related to the provided context, but it does not fully answer the question or provide extensive detail.
```

confirmed that this works fine for gpt-4